src/main/resources/logback.xml

```diff
@@ -0,0 +1,43 @@
+<configuration>
+    <appender name="JSON-STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <target>System.out</target>
+        <encoder class="net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder">
+            <providers>
```

Comment:
Why is it required to define the providers? can't we use the existing [default layout for JSON](https://mathieularose.com/logback-json/)?

```
<appender name="stdout" class="ch.qos.logback.core.ConsoleAppender">
        <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
            <layout class="ch.qos.logback.contrib.json.classic.JsonLayout">
                <timestampFormat>yyyy-MM-dd'T'HH:mm:ss.SSSX</timestampFormat>
                <timestampFormatTimezoneId>Etc/UTC</timestampFormatTimezoneId>

                <jsonFormatter class="ch.qos.logback.contrib.jackson.JacksonJsonFormatter">
                    <prettyPrint>true</prettyPrint>
                </jsonFormatter>
            </layout>
        </encoder>
    </appender>
```

Reply:
I would not say it's default but an additional package as named contrib, I used the https://github.com/logstash/logstash-logback-encoder which is used by other projects and we know it has good support on general JSON encoding. So not seeing much value in changing this.

Reply:
Ohh okay, I missed that part. My intention was not to use an extra dependency just to get the JSON output. But it seems there is no other way.